vault/README.md

```diff
@@ -12,60 +12,66 @@ The Vault check is included in the [Datadog Agent][2] package.
 
 #### Prerequisites
 
+This OpenMetrics-based integration has a latest version (`use_openmetrics`: true) and a legacy version (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends upgrading to the latest version. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][22].
+
 1. Ensure you have enabled [Prometheus metrics in the Vault configuration][3].
 
-2. For the Vault check to work properly, you need to either enable unauthenticated access to Vault metrics (Vault 1.3.0+) or provide a Vault client token:
+   * [Metrics starting with `vault.replication.fetchRemoteKeys`, `vault.replication.merkleDiff`, and `vault.replication.merkleSync`](#data-collected) are not reported unless the replication is in an unhealthy state. 
+   * Versions 3.4.0 and later of this check use [OpenMetrics][21] for metric collection, which requires Python 3. For hosts that are unable to use Python 3, or if you would like to use a legacy version of this check, set the value of `use_openmetrics` to `false` in the configuration.
```

Comment:
I think I would integrate this paragraph with the first one about latest vs legacy, as it's otherwise a bit redundant. We should also probably try to avoid using the bare OpenMetrics name to refer to the V2 behavior.

Reply:
```suggestion
```